package.json

```diff
@@ -28,13 +28,13 @@
   },
   "scripts": {
     "build": "rollup -c",
-    "build-test": "rollup -c test/rollup.unit.config.js -w",
+    "build-test": "rollup -c test/rollup.unit.config.js",
     "build-uglify": "rollup -c && uglifyjs build/three.js -cm --preamble \"// threejs.org/license\" > build/three.min.js",
     "build-closure": "rollup -c && java -jar node_modules/google-closure-compiler/compiler.jar --warning_level=VERBOSE --jscomp_off=globalThis --jscomp_off=checkTypes --externs utils/build/externs.js --language_in=ECMASCRIPT5_STRICT --js build/three.js --js_output_file build/three.min.js",
     "dev": "concurrently --names \"ROLLUP,HTTP\" -c \"bgBlue.bold,bgGreen.bold\" \"rollup -c -w -m inline\" \"serve --port 8080\"",
     "start": "npm run dev",
     "lint": "eslint src",
-    "test": "qunit test/unit/three.source.unit.js",
+    "test": "npm run build-test && qunit test/unit/three.source.unit.js > report.txt",
```

Comment:
` > report.txt`.

I think i would prefer to remove this and output the result to the console like before.

Reply:
Actually that was from my test case. This shouldn't have been committed. 
Thanks